examples/runCfhtQuickTest.sh

```diff
@@ -7,12 +7,45 @@ CAMERA=CfhtQuick
 CONFIG_FILE="${PRODUCT_DIR}/config/cfhtConfig.py"
 MAPPER=lsst.obs.cfht.MegacamMapper
 
-"${PRODUCT_DIR}/examples/processData.sh" \
-    -c "$CAMERA" \
-    -m "$MAPPER" \
-    -v "$VALIDATION_DATA_DIR" \
-    -f "$CONFIG_FILE" \
-    -- "$@"
-"${PRODUCT_DIR}/examples/validateRepo.sh" \
-    -c "$CAMERA" \
-    -- "$@"
+print_error() {
+    >&2 echo "$@"
+}
+
+DOPROCESS=true
+DOVERIFY=true
+
+usage() {
+    print_error
```

Comment:
This down to personal preference but you could replace the multiple `print_error` calls with.
```
print_error "$(cat <<-EOF

EOF
)"
```

Reply:
I'd prefer to leave this as is since it's the same as in other scripts in this repo.  I'm happy to file a ticket to update the syntax in the future.